tests/integration/target-prefixes.network

```diff
@@ -1 +1 @@
-netconf
\ No newline at end of file
+yang
```

Comment:
This change is not required and the file can be empty Refer https://github.com/ansible/ansible-zuul-jobs/blob/10fd68e938f5a4b369b6f60b623dc467cda26bf5/zuul.d/jobs.yaml#L1918